runtime/fastly/host-api/error_numbers.msg

```diff
@@ -65,6 +65,7 @@ MSG_DEF(JSMSG_KV_STORE_KEY_ACME,                               0, JSEXN_TYPEERR,
 MSG_DEF(JSMSG_KV_STORE_KEY_RELATIVE,                           0, JSEXN_TYPEERR, "KVStore key can not be '.' or '..'")
 MSG_DEF(JSMSG_KV_STORE_PUT_CONTENT_STREAM,                     0, JSEXN_TYPEERR, "Content-provided streams are not yet supported for streaming into KVStore")
 MSG_DEF(JSMSG_KV_STORE_PUT_OVER_30_MB,                         0, JSEXN_TYPEERR, "KVStore value can not be more than 30 Megabytes in size")
+MSG_DEF(JSMSG_KV_STORE_PUT_OPTIONS,                            1, JSEXN_TYPEERR, "KVStore insert option: {0}")
```

Comment:
This would throw the error `KVStore insert options: if generation match`. Maybe we can refine this to be the generic `api_error`, which StarlingMonkey has been migrating to with a nicer message like: `KVStore insert: "gen" must be a number` or something like that.

Reply:
good idea, i changed this in 8e0616f99a3d6b9426ad675b085005cdd960c50c